CLAUDE.md

```
@@ -1,13 +1,21 @@
 
-This file provides guidance to Claude Code (claude.ai/code) when working with code in this repository.
+This file provides guidance to Claude and other AI models and agents when working with the code in this repository.
 
 ## Project Overview
 
-GOModel is a high-performance AI gateway written in Go that routes requests to multiple LLM providers (OpenAI, Anthropic, Google Gemini). It's designed as a drop-in replacement for LiteLLM with superior concurrency and strict type safety.
+GOModel is a high-performance AI gateway written in Go that routes requests to multiple LLM providers (OpenAI, Anthropic, Google Gemini). It is designed as a drop-in replacement for LiteLLM, offering superior quality, speed, concurrency, and strict type safety.
 
 **Module name:** `gomodel`
 **Go version:** 1.24.0
+**GitHub link:** https://github.com/ENTERPILOT/GOModel
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)